test/functionalTest/cases/3122_location_metadata_issues/get_location_attr_with_db_creation.test

```diff
@@ -0,0 +1,152 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Create an inconsistent entity with a location attribute and no metadata directly in DB.
+GET the resulting entity both with NGSIV1 and NGSV2.
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Create E1 directly in DB
+# 02. Get E1 with NGSIV1
+# 03. Get E1 with NGSIV2
+
```

Comment:
I would suggest to use more precise steps (including echo messages) in this test. For instance:

```
# 01. Create E1 directly in DB with location GeoJSON field but without location metadata nor geo:xxx type
# 02. Get E1 with NGSIV1 with location metadata
# 03. Get E1 with NGSIV2 without location metadata
```

Reply:
Fixed in c76622d